addon/components/frost-url-input.js

```diff
@@ -0,0 +1,106 @@
+/*
+ Based on the textarea field, the URL input field consists of an input field and a button.
+ The component is used to verify that the provided URL is both valid and accessible.
+
+ After clicking the 'Test' button, the entered URL is loosely verified to be correctly formatted.
+ An AJAX GET request is then sent (with JSONP to overcome cross-domain problems) to the destination URL.
+ The response is checked with any status code between 100 and 500, except 404, deemed a success.
+
+ Should MIME Checking be enabled and the Type set to 'Text',
+ an error will be thrown when using the Chrome browser
+ */
+
+import Ember from 'ember'
+const {Component} = Ember
+import layout from '../templates/components/frost-url-input'
+
+// Override Lint checking problem for undefined $
+/* global $ */
+
+export default Component.extend({
+  // ==========================================================================
+  // Dependencies
+  // ==========================================================================
+
+  // ==========================================================================
+  // Properties
+  // ==========================================================================
+
+  classNames: ['frost-url-input'],
+
+  error: false,
+  isLoading: false,
+  success: false,
+  undetermined: false,
+  urlFormatError: false,
+  placeholder: 'Enter URL',
```

Comment:
This looks like a good candidate for [ember-prop-types](https://github.com/ciena-blueplanet/ember-prop-types).